core/src/main/java/com/netflix/conductor/core/execution/mapper/ExclusiveJoinTaskMapper.java

```diff
@@ -51,7 +51,9 @@ public List<TaskModel> getMappedTasks(TaskMapperContext taskMapperContext) {
         TaskModel joinTask = taskMapperContext.createTaskModel();
         joinTask.setTaskType(TaskType.TASK_TYPE_EXCLUSIVE_JOIN);
         joinTask.setTaskDefName(TaskType.TASK_TYPE_EXCLUSIVE_JOIN);
-        joinTask.setStartTime(System.currentTimeMillis());
+        final long currentTimeMillis = System.currentTimeMillis();
+        joinTask.setScheduledTime(currentTimeMillis);
```

Comment:
`taskMapperContext.createTaskModel()` sets the `scheduledTime` already. I missed that during our discussion. I don't see a need to scheduledTime in the TaskMapper implementations. 

Its possible that queueWaitTime is a non-zero value for synchronous system tasks, but as long as its not below zero, we are good.